examples/gradient-circle.js

```diff
@@ -30,8 +30,8 @@ const Example = () => {
           strokeWidth="6"
           strokeLinecap="round"
           strokeColor={{
-            '100%': '#87d068',
-            '0%': '#108ee9',
+            '0%': '#87d068',
+            '100%': '#108ee9',
```

Comment:
No need to modify example. This order is by intended.

Reply:
The demo would fail. 

<img width="345" alt="屏幕快照 2019-08-06 下午4 02 24" src="https://user-images.githubusercontent.com/12122021/62522133-a3269a80-b863-11e9-9bf1-7bdbb7d973d5.png">

Actually that indented config works because of the bug this pr is trying to fix.

Reply:
Yes. That's why we use this order of object define. We should handle this.